actions/arteria_delivery_service.py

```diff
@@ -128,7 +128,11 @@ def _get_from_server(self, url):
     @staticmethod
     def parse_stage_order_ids_from_response(response):
         projects_and_staging_order_ids = response['staging_order_ids']
-        staged_runfolders = list(map(lambda staged_path: staged_path.get('path',None), response['staged_data']))
+        staged_data = response.get('staged_data',None)
+        if staged_data is not None:
```

Comment:
I'd suggest using `if staged_data` here instead. I'd say it is more "pythonic". In general people will use the "truthiness" principles in python for these type of things. http://csparpa.github.io/blog/2015/02/python-truthiness.html